package.json

```diff
@@ -44,48 +44,48 @@
     "url": "https://github.com/watson-developer-cloud/node-sdk/issues"
   },
   "devDependencies": {
-    "@types/csv-stringify": "^1.4.1",
-    "@types/extend": "^3.0.0",
-    "@types/file-type": "^5.2.1",
-    "@types/is-stream": "^1.1.0",
-    "@types/node": "^8.0.47",
-    "@types/request": "^2.0.7",
-    "browserify": "^14.5.0",
-    "concat-stream": "^1.5.1",
-    "dependency-lint": "^5.0.1",
-    "eslint": "^4.12.1",
-    "eslint-config-google": "^0.9.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-prettier": "^2.3.1",
-    "jsdoc": "^3.5.5",
-    "memory-fs": "^0.4.1",
-    "mocha": "^3.5.3",
-    "nock": "^9.1.3",
-    "nyc": "^11.4.0",
-    "object.assign": "^4.0.4",
-    "prettier": "^1.8.2",
-    "sinon": "^4.1.2",
-    "typescript": "^2.6.2",
-    "uglify-es": "^3.2.0",
-    "watchify": "^3.7.0",
-    "wav": "^1.0.0",
-    "webpack": "^3.9.1"
+    "browserify": "~14.5.0",
+    "concat-stream": "~1.6.0",
+    "dependency-lint": "~5.0.1",
+    "eslint": "~4.12.1",
+    "eslint-config-google": "~0.9.1",
+    "eslint-config-prettier": "~2.9.0",
+    "eslint-plugin-node": "~5.2.1",
+    "eslint-plugin-prettier": "~2.3.1",
+    "jsdoc": "~3.5.5",
+    "memory-fs": "~0.4.1",
+    "mocha": "~4.0.1",
+    "nock": "~9.1.4",
+    "nyc": "~11.4.0",
+    "object.assign": "~4.0.4",
+    "prettier": "~1.9.1",
+    "sinon": "~4.1.2",
+    "typescript": "~2.6.2",
+    "uglify-es": "~3.2.1",
+    "watchify": "~3.9.0",
+    "wav": "~1.0.0",
+    "webpack": "~3.10.0"
   },
   "dependencies": {
-    "async": "^2.6.0",
-    "buffer-from": "^0.1.1",
+    "@types/csv-stringify": "~1.4.1",
+    "@types/extend": "~3.0.0",
+    "@types/file-type": "~5.2.1",
+    "@types/is-stream": "~1.1.0",
+    "@types/node": "~8.0.55",
+    "@types/request": "~2.0.7",
+    "async": "~2.6.0",
+    "buffer-from": "~0.1.1",
     "cookie": "~0.3.1",
-    "csv-stringify": "~1.0.2",
+    "csv-stringify": "~2.0.0",
```

Comment:
I'm a little bit worried about this 😱  it's a major release. Can you check if they changed something that may break users trying to use json to create a classifier?

Reply:
Yeah, it actually broke the `node 4` build. I am reverting this one back to `1.0.2`.